var/spack/repos/builtin/packages/moab/package.py

```diff
@@ -67,13 +68,35 @@ def install(self, spec, prefix):
         options = [
             '--prefix=%s' % prefix,
             '--enable-optimize',
-            '--enable-tools',
+            '--disable-tools',
+            '--disable-mbconvert',
+            '--disable-hexmodops',
+            '--disable-vtkMOABReader',
+            '--disable-mbsize',
+            '--disable-mbskin',
+            '--disable-mbtagprop',
+            '--disable-mbmem',
+            '--disable-spheredecomp',
+            '--disable-mbsurfplot',
+            '--disable-mbpart',
+            '--disable-dagmc',
+            '--disable-gsets',
+            '--disable-mbmerge',
+            '--disable-mbdepth',
+            '--disable-mbcoupler',
+            '--disable-mcnpmit',
+            '--disable-refiner',
+            '--disable-h5mtools',
+            '--disable-mbcslam',
+            '--disable-mbquality',
+            '--disable-ahf',
+            '--disable-mbumr',
+            '--disable-imesh',
```

Comment:
Because --disable-tools (which was _supposed_ to have that effect) was apparently missing some...because the tools are needed to install the library and because they create additional dependencies I don't want to add at this time. I work with MOAB developers too. I have adopted a strategy of minimal installs for which we can slowly add functionality.